.github/workflows/continuous-integration.yml

```diff
@@ -7,38 +7,28 @@ jobs:
   phpunit-mariadb:
     name: "PHPUnit with MariaDB: ${{ matrix.mariadb-version }}, PHP: ${{ matrix.php-version }}, extension: ${{ matrix.extension }}"
     runs-on: ubuntu-latest
-    # If true, allow the job to fail:
-    continue-on-error: ${{ matrix.experimental }}
     strategy:
       # If true, stop jobs if a required job fails:
       fail-fast: false
       matrix:
         # Define jobs for all combinations of php, mariadb and extension, up to "include"
         # Tests will be performed for each combination
-        # These can fail if "experimental" is true. Currently all must pass
-        php-version: ["5.4", "5.5", "5.6", "7.0", "7.1"]
+        php-version: ["5.4", "5.5", "5.6", "7.0", "7.1", "7.4"]
         mariadb-version: ["10.3"]
         extension: ["pdo_mysql"]
-        experimental: [false]
         include:
-          # Define jobs for individual combinations to be tested
-          # These can fail if "experimental" is true. Currently all can fail
-          - php-version: "7.2"
-            mariadb-version: "10.3"
-            extension: "pdo_mysql"
-            experimental: true
-          - php-version: "7.3"
-            mariadb-version: "10.3"
-            extension: "pdo_mysql"
-            experimental: true
+          - php-version: "5.4"
+            composer-json: "composer.json"
```

Comment:
let's give it a go! :)

Reply:
I don't think it can, atleast not without using excludes.

Both these result in two PHP 7.4 jobs being generated, one with `composer.json` and one with `composer-7.4.json` being used.

```
# Define jobs for all combinations of php, mariadb and extension, up to "include"
# Tests will be performed for each combination
php-version: ["5.4", "5.5", "5.6", "7.0", "7.1", "7.4"]
mariadb-version: ["10.3"]
extension: ["pdo_mysql"]
composer-json: ["composer.json"]
include:
  - php-version: "7.4"
    composer-json: "composer-7.4.json"
```
action run for above conf: https://github.com/gregcorbett/gocdb/actions/runs/6532629042

```
# Define jobs for all combinations of php, mariadb and extension, up to "include"
# Tests will be performed for each combination
php-version: ["5.4", "5.5", "5.6", "7.0", "7.1", "7.4"]
mariadb-version: ["10.3"]
extension: ["pdo_mysql"]
composer-json: ["composer.json"]
include:
  - php-version: "7.4"
    mariadb-version: "10.3"
    extension: "pdo_mysql"
    composer-json: "composer-7.4.json"
```
action run for above conf: https://github.com/gregcorbett/gocdb/actions/runs/6532671013